cmd/agent/main.go

```diff
@@ -58,9 +58,9 @@ func main() {
 	var controlPlaneConfigSecretName string
 	var controlPlaneConfigSecretNamespace string
 
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8084", "The address the metric endpoint binds to.")
-	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8085", "The address the probe endpoint binds to.")
-	flag.StringVar(&controlPlaneConfigSecretName, "control-plane-cluster", "control-plane-cluster", "The name of the secret with the control plane client configuration")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
why the change of port?
Won't this conflict with the controller if running that locally too?

Reply:
yes, my thoughts were to set the values in the makefile and deployment.yaml, and leave the hardcoded values sane

Reply:
OK.
So as it stands, the metrics-bind-address default value is 8080 for the ingress agent and the controller, while the syncer is 8084.
The health-probe-bind-address default is 8081 for ingress, 8081 for controller and 8085 for syncer.
Should the defaults be changed for the ingress agent or controller so they don't clash,
or should the make target(s) for either overwrite those args?